src/module/item/entity.js

```diff
@@ -107,7 +110,8 @@ export class OseItem extends Item {
         default: "melee",
       }).render(true);
       return true;
-    } else if (itemData.missile && !isNPC) {
+    }
+    if (itemData.missile && !isNPC) {
```

Comment:
I've been staring at this one for a bit. Isn't this changing behavior to always choose missile no matter what you said in the dialog?

Reply:
From what I understand, it's cutting the `if... else if...` into two `if...` blocks If the first `if...` works, we're returning on line 112.

The fallthrough state is `attack` or `melee`, depending on if the Actor is an NPC.

I feel like we might be able to rewrite the logical stuff here to be a little less brain-intensive.

Reply:
the return is what I was missing, thanks